CAKE-autorate.sh

```diff
@@ -8,13 +8,22 @@
 
 # Possible performance improvement
 export LC_ALL=C
-export TZ=UTC
+# export TZ=UTC
 
 trap cleanup_and_killall INT TERM EXIT
 
+# Format log entries - includes timestamp on each line
+# Entries should contain their own newline, if necessary
+log_msg()
+{
```

Comment:
```suggestion
{
    printf '%(%F %H:%M:%S)T %s' -1 "$1"
```

Reply:
I try to avoid $( ) wherever possible because it is costly and also to use bash inbuilt functions wherever possible because this consumes less CPU cycles.